.kitchen.yml

```diff
@@ -5,6 +5,7 @@ driver:
 provisioner:
   name: chef_zero
   require_chef_omnibus: 12.19.36
+  #require_chef_omnibus: 13.8.5
```

Comment:
You're installing using chef 12 everywhere?

We will be dropping 12 support soon

Reply:
Yeah so was planning to do that in another PR and this line got added here accidentally. But we can get that included here.